docs/api-assetmanager.md

```diff
@@ -0,0 +1,161 @@
+
+<p align="center">
+    <h1 align="center">Api Library AssetManager</h1>
+</p>
+
+The **AssetManager** is a library that allows you to **generate the configuration of all your css and javascript assets easily**, and then generate your html code.
+
+### setAppendTimestamp:
```

Comment:
API docs should stay in phpdoc. There's no need to have these in readme.

Reply:
Move docs, Done.